channels/layers.py

```diff
@@ -173,13 +176,20 @@ def non_local_name(self, name):
 
 
 class InMemoryChannelLayer(BaseChannelLayer):
-    """
-    In-memory channel layer implementation for testing purposes.
-    """
+    '''
+    Our own in memory layer
```

Comment:
Did you mean to change the docstring?

Reply:
No that was left in there from testing I changed it to "In-memory channel layer implementation" since it can be used for more than testing now